repository-ydb-v1/src/main/java/tech/ydb/yoj/repository/ydb/YdbSpliterator.java

```diff
@@ -116,24 +116,33 @@ private QueueValue<V> poll() {
     @Override
     // (stream thread) called from stream engine for get new value.
     public boolean tryAdvance(Consumer<? super V> action) {
```

Comment:
question: What should we do if spliterator closed ?

Reply:
I think we should just return `false` from `tryAdvance`. Java's `Spliterator` interface declares no exceptions except for NPE than can be thrown from `Spliterator.tryAdvance()`